### PR TITLE
Fixing `xxh3_128_with_secret` function.

### PR DESCRIPTION
**Pull request by farhadi**

### CODE REVIEW

File: src/xxh3.rs [hunk 1]
```diff
@@ -890,6 +890,7 @@ pub fn xxh3_128_with_seed(input: &[u8], seed: u64) -> u128 {
 
 #[inline]
 ///Returns 128 hash for provided input using custom secret.
-pub fn xxh3_128_with_secret(input: &[u8]) -> u128 {
-    xxh3_128_internal(input, 0, &DEFAULT_SECRET, xxh3_128_long_with_secret)
+pub fn xxh3_128_with_secret(input: &[u8], secret: &[u8]) -> u128 {
+    debug_assert!(secret.len() >= SECRET_SIZE_MIN);
```
Comment (DoumanAsh) [inline src/xxh3.rs:894]: This is performed in `xxh3_128_internal` on line 847 though?

Comment (farhadi) [reply src/xxh3.rs:894]: Ok, then it can also be removed for `xxh3_64_with_secret`?

**Comment by DoumanAsh** [comment]
Thank you for noticing it. I do not use custom secret myself and didn't notice that I forgot to finish up this function!

**Comment by DoumanAsh** [comment]
I've yanked 0.8.0, it is not valid for use.
I pushed new version 0.8.1, and consider it as not breaking change due to 0.8.0 no longer being available.

This is not standard practice, but I want to keep major/minor version in correspondence with C implementation

**Pull request merged**